components/country/overview.js

```diff
@@ -0,0 +1,171 @@
+              values={{ circumventionTools }}
+            />
+          </NwInterferenceStatus> */}
+          <NwInterferenceStatus width={[1, 1/2]} color={imCount && 'yellow9'}>
+            <NettestGroupInstantMessaging size={32} />
+            <FormattedMessage
+              id={ getStatus(imCount, 'Country.Overview.NwInterference.IM') }
+              values={{ imCount }}
+            />
+          </NwInterferenceStatus>
+          <NwInterferenceStatus width={[1, 1/2]} color={blockedWebsitesCount && 'indigo5'}>
+            <NettestGroupWebsites size={32} />
+            <FormattedMessage
+              id={ getStatus(blockedWebsitesCount, 'Country.Overview.NwInterference.Websites') }
+              values={{ blockedWebsitesCount }}
+            />
+          </NwInterferenceStatus>
+        </Flex>
+      </BoxWithTitle>
+      <Heading h={4}>
+        <FormattedMessage id='Country.Overview.Heading.TestsByClass' />
+      </Heading>
+      <TestsByGroup
+        fetchTestCoverageData={fetchTestCoverageData}
+        testCoverage={testCoverage}
+        networkCoverage={networkCoverage}
+      />
+      <BoxWithTitle title={<FormattedMessage id='Country.Overview.FeaturedResearch' />}>
+        {
+          (featuredArticles.length === 0)
+            ? <FormattedMessage id='Country.Overview.FeaturedResearch.None' />
+            : <ul>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/cuba-referendum/'
+                  title='Cuba blocks independent media amid 2019 constitutional referendum'
+                />
+              </li>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/ooni-software-development-guidelines/'
+                  title='OONI Software Development Guidelines'
+                />
+              </li>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/ooni-probe-android-200-incident/'
+                  title='OONI Probe Android 2.0.0 Incident Report'
+                />
+              </li>
+              <li>
+                <FeaturedArticle
+                  link='https://ooni.io/post/venezuela-blocking-wikipedia-and-social-media-2019/'
+                  title='From the blocking of Wikipedia to Social Media: Venezuela&quote;s Political Crisis'
+                />
+              </li>
+            </ul>
+        }
+      </BoxWithTitle>
+      {/* Highlight Box */}
+      {/* <SummaryText> */}
+      <SummaryText>
+        <FormattedMessage
+          id='Country.Overview.SummaryTextTemplate'
+          values={{
+            measurementCount,
+            countryName,
+            startDate: new Date(startDate).toLocaleDateString(),
+            networkCovered,
+            totalNetworks: 9999,
+            mostBlockedASNname: 'Blocker McBlocky Inc.',
+            mostBlockedASN: 'AS5h013'
```

Comment:
I guess this is a placeholder. We should drop it and populate it from real data.

Reply:
Replaced with most 'measured' network info as per copy.